list.json

```diff
@@ -784,6 +784,14 @@
     "name": "Hastie",
     "github": "mkaz/hastie"
   },
+  {
+    "name": "Haunt",
+    "website": "https://dthompson.us/projects/haunt.html",
+    "language": "Scheme",
+    "description": "A simple, functional, hackable static site generator that gives authors the ability to treat websites as Scheme programs.",
+    "license": "GPL-3.0-or-later",
+    "is": "static site generator",
+    "extensible": true
```

Comment:
Missing a }, here?

Reply:
Indeed. Thanks, I'll fix that…

Reply:
fixed in 564a57e6d23c63e268e4b91ffe2f0e48dccbf01d